process/smartContract/process.go

```diff
@@ -481,10 +528,15 @@ func (sc *scProcessor) computeTotalConsumedFeeAndDevRwd(
 	}
 
 	senderInSelfShard := sc.isSelfShard(tx.GetSndAddr())
-	consumedGas := safeSubUint64(tx.GetGasLimit(), vmOutput.GasRemaining)
+	consumedGas, err := safeSubUint64(tx.GetGasLimit(), vmOutput.GasRemaining)
+	log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd")
```

Comment:
You need a third parameter here, otherwise computeTotalConsumedFeeAndDevRwd won't be displayed. Add more info as to easily pin-point the errors in case they appear.

Reply:
done